chain/network/src/peer_manager/testonly.rs

```diff
@@ -136,7 +136,7 @@ impl ActorHandler {
     pub fn peer_info(&self) -> PeerInfo {
         PeerInfo {
             id: PeerId::new(self.cfg.node_key.public_key()),
-            addr: self.cfg.node_addr.clone(),
+            addr: self.cfg.node_addr.as_ref().map(|a| **a),
```

Comment:
I’m seeing a fair number of these around the code – would it make sense to add a convenience getter to the config?

Reply:
I like it explicit, at least as of today.